selftests/functional/plugin/test_logs.py

```diff
@@ -3,7 +3,7 @@
 from avocado.core import exit_codes
 from avocado.utils import process
 
-from .. import AVOCADO, TestCaseTmpDir
+from ... import AVOCADO, TestCaseTmpDir
```

Comment:
This could be out of this PR, but I will close my eyes here.

Reply:
Well, it should have been on the that specific PR, but I missed it.  Then, this PR adds a new test for this plugin, so I needed to move this file for consistency.